guides/source/testing.md

```diff
@@ -761,8 +761,7 @@ and also ensuring that the right response body has been generated.
 
 The `get` method kicks off the web request and populates the results into the `@response`. It can accept up to 6 arguments:
 
-* The action of the controller you are requesting.
-  This can be in the form of a string or a route (i.e. `articles_url`).
```

Comment:
OK. I wanted to remove reference to string vs route, because I thought it was a little confusing. Isn't articles_url a method that returns a string?